### PR TITLE
nv.models.multiBarHorizontalChart

### PR DESCRIPTION
**Pull request by robrez**
Created nvd3-multi-bar-horizontal-chart in development branch

### CODE REVIEW

File: nvd3-multi-bar-horizontal-chart/nvd3-multi-bar-horizontal-chart.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!--
+    @license
+    Copyright (c) 2015 Renato Utsch. All rights reserved.
+    This code may only be used under the BSD style license found at http://renatoutsch.github.io/LICENSE.txt
+-->
+<link rel="import" href="../nvd3-multi-bar-chart/nvd3-multi-bar-chart.html">
+
+<!--
+This element is a polymer wrapper for the NVD3 multi bar chart.
```
Comment (RenatoUtsch) [inline nvd3-multi-bar-horizontal-chart/nvd3-multi-bar-horizontal-chart.html]: You forgot to change the name here (add an "horizontal") hehe

File: nvd3-multi-bar-horizontal-chart/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+nvd3-multi-bar-horizontal-chart
+=======================
```
Comment (RenatoUtsch) [inline nvd3-multi-bar-horizontal-chart/README.md]: Missing some ====

**Comment by robrez** [comment]
Corrected the problems you mentioned

**Comment by RenatoUtsch** [comment]
Thanks for the help!

**Pull request merged**